lib/identity/client.js

```diff
@@ -123,24 +128,52 @@ class Client extends PartialClient {
     const data = {
       secretValue: secret.secretValue,
     }
+    let timestamp = Date.now().toString()
     const meta = {
       secretType: secret.secretType,
       secretName: secret.secretName,
       description: secret.description,
-      version: 'latest',
+      version: timestamp,
```

Comment:
@efabens I assume this is what you guys settled instead of version: 'latest'; however, I don't think this provides much value over our created at timestamp since plain_meta is string based and not order-able

Reply:
Yeah doesn't provide value for elastic search ordering but provides value as a piece of data that is constant that a consumer could use. It was between using timestamp or not having a value

Reply:
@efabens we do want to be able to show users both in the cli and the ui a version NUMBER, I'm fine if we use timestamps under the hood but keep that constraint in mind

Reply:
@galxy25 given that we can delete records I don't think that we can guarantee that an iterating number can be trusted to always be correct. If in a user interacting situation we want an easy way to list out version numbers than I think the UI should order and create temporary values that match versions. For an automated system where they know a particular version is wanted than the timestamp provides a constant that can be trusted